proprietary/tokens/src/common/ams/typography.tokens.json

```diff
@@ -0,0 +1,61 @@
+{
+  "ams": {
+    "common": {
+      "typography": {
+        "body-text": {
+          "font-family": { "value": "{ams.brand.typography.font-family.body-text}" },
```

Comment:
We might need to look at the bundler for the tokens here, because everything is bundled. So you're creating:

`--ams-brand-typography-font-family` and `--ams-common-typography-body-text-font-family`

Do we need both and how is it more semantic?